### ISSUE TITLE
UI Broken and Odd Behavior

### ISSUE THREAD

**Issue by behezare** [GrapheneOS/os-issue-tracker#2576]
This is happening on a Pixel 7 device. It first started last Sunday, October 8th. There are many different issues with my phone going on all at once. This began after some freezing glitches would occur every once in a while, then while a particularly long one happened that was about 10 to 20 seconds long, in a separate account from my owner account I restarted my phone and this all started happening upon powering up again. My settings have all been reset but most of my apps still behave the same as before.

The notification tray is broken and does not display any notifications as well as being unable to swipe down to access the quick settings. Swiping down to try to get to the notifications just shows a notification tray that permanently displays no notifications and trying to swipe down again does not display the quick settings menu although it still looks like the swipe is registering. I can go into notification history in settings and see new notifications, but they all show up under recently dismissed immediately.

Gestures controls are broken and swiping from the sides to go back does nothing even when I try increasing the sensitivity to maximum. One weird thing actually is that when the problems began a little back arrow showed up in the bottom left that allows me to go back that wasn't there before. Swiping up to go home and holding it to see recent apps still works. When I switch it to 3-button navigation the buttons on the bottom show up but the back button is still the only one that does anything. The others give feedback when pressed but don't actually do anything. The recent apps holds no history of any app other than the one I'm currently using.

On top of this, I am unable to access many settings throughout the phone. Long pressing on an app and selecting app info makes the settings pop up for a split second but then nothing happens. When I try selecting some options in the settings menu like Passwords and accounts it does the same behavior as before where it registers the taps but doesn't actually let me in.

Some more info I think might be relevant: Google play services and stuff like that was enabled on the other account I use as it was my school/work account. I also remember installing an app with some pretty invasive permissions required onto that account because I needed it for my apartment not too long before I started having issues. Because of this, I also unsynced my account with my owner account when they had been synced for a long time before.

I managed to change into the alpha branch and update the device thinking the upgrade to android 14 would fix everything, but it changed nothing. Someone on the matrix server suggested posting a video of the issue so here it is:

https://github.com/GrapheneOS/os-issue-tracker/assets/148404638/379cc86b-d236-4477-9941-b5b6034baff9

**Comment by thestinger** [comment]
If you're using a non-default launcher, try the default launcher. Clearing the default launcher's data would be a worthwhile thing to try if you're using it.

**Comment by behezare** [comment]
> If you're using a non-default launcher, try the default launcher. Clearing the default launcher's data would be a worthwhile thing to try if you're using it.

I am using the stock launcher. I actually tried installing simple launcher and using it to see if that fixed things but it didn't and behaved weirdly. It was as though using the launcher counted as another app I could just exit out of. I also tried deleting the default launchers data by going through storage>apps>Launcher>clear storage since I can't access any app permissions or settings, but all that did was remove all the apps from my home screen.
